### ISSUE TITLE
Accessibility: iframe title attribute

### ISSUE THREAD

**Issue by afercia** [swissspidy/oEmbed-API#42]
For accessibility, iframes should have a title attribute to indicate (at least) their type of content. See for example the markup returned by Twitter:

![Twitter embed]
(https://cldup.com/aUPcAjBC11.png)

**Comment by swissspidy** [comment]
Didn't know that, thanks for the heads-up!

Ideas for the title attribute:

* Embedded post
* WordPress post
* Embed: <the post title>

Any better ideas? :)

**Comment by swissspidy** [comment]
I used `Embedded WordPress Post` for now. I'll reopen this if doesn't feel right.

Thanks again.
